src/react-components/room/RoomSidebar.js

```diff
@@ -135,7 +136,59 @@ export function SceneInfo({ accountId, scene, showAttributions, canChangeScene,
   );
 }
 
-export function RoomSidebar({ room, accountId, onClose, canEdit, onEdit, onChangeScene }) {
+function DebugSceneDropzone({ hubChannel }) {
+  const [lastBlobUrl, setLastBlobUrl] = useState();
+  const [dropping, setDropping] = useState(false);
+
+  const handleDragEnter = e => {
+    e.preventDefault();
+    setDropping(true);
+  };
+  const handleDragLeave = e => {
+    e.preventDefault();
+    setDropping(false);
+  };
+
+  const handleDragOver = e => {
+    e.preventDefault();
+    e.dataTransfer.dropEffect = "move";
+  };
+
+  const handleDrop = e => {
+    e.preventDefault();
```

Comment:
Not critical but it's confusing that the object dropping stops working during scene debug. I think we can stop the event bubbling propagation here using `e.stopPropagation()` and we can keep the top object drop behavior untouched.

Reply:
Yeah removing this from this PR, I ended up doing it in a different PR